src/main/resources/com/waytta/clientinterface/LocalBatchClient/config.jelly

```diff
@@ -4,26 +4,21 @@
   <f:entry title="Batch Size" field="batchSize">
     <f:textbox default="50%" />
   </f:entry>
-
-  <f:invisibleEntry>
-    <input name="clientInterface" type="hidden" value="local_batch"/>
-  </f:invisibleEntry>
   
   <f:entry title="Target" field="target">
     <f:textbox />
   </f:entry>
   
   <f:entry title="Target Type" field="targetType">
     <select name="targetType">
-      <f:option value="glob" selected="${instance.getTargettype() == 'glob'}">glob</f:option>
-      <f:option value="pcre" selected="${instance.getTargettype() == 'pcre'}">pcre</f:option>
-      <f:option value="list" selected="${instance.getTargettype() == 'list'}">list</f:option>
-      <f:option value="grain" selected="${instance.getTargettype() == 'grain'}">grain</f:option>
-      <f:option value="pillar" selected="${instance.getTargettype() == 'pillar'}">pillar</f:option>
-      <f:option value="nodegroup" selected="${instance.getTargettype() == 'nodegroup'}">nodegroup</f:option>
-      <f:option value="range" selected="${instance.getTargettype() == 'range'}">range</f:option>
-      <f:option value="compound" selected="${instance.getTargettype() == 'compound'}">compound</f:option>
+      <f:option value="glob" selected="${instance.getTargetType() == 'glob'}">glob</f:option>
```

Comment:
You do not need any of this. Just use `f:select` and have the descriptor supply choices.